backend/homework/5/src/main/java/log/Log.java

```diff
@@ -0,0 +1,24 @@
+package log;
+
+import java.util.logging.Level;
+public class Log {
+    private Log()
+    {
+
+    }
+    private static final java.util.logging.Logger LOGGER = java.util.logging.Logger.getLogger(Log.class.getName());
```

Comment:
this part should be in the import statement `java.util.logging.Logger`
    `private static final Logger LOGGER = Logger.getLogger(Log.class.getName());`